pkg/tsdb/cloudwatch/query_transformer.go

```diff
@@ -104,10 +99,124 @@ func (e *cloudWatchExecutor) transformQueryResponsesToQueryResult(cloudwatchResp
 			queryResult.ErrorString = "Cloudwatch GetMetricData error: Too many datapoints requested - your search has been limited. Please try to reduce the time range"
 		}
 
+		eq, err := json.Marshal(executedQueries)
+		if err != nil {
+			return nil, fmt.Errorf("could not marshal executedString struct: %w", err)
+		}
+
+		link, err := buildDeepLink(refID, requestQueries, executedQueries, startTime, endTime)
+		if err != nil {
+			return nil, fmt.Errorf("could not build deep link: %w", err)
+		}
+
+		for _, frame := range frames {
+			frame.Meta = &data.FrameMeta{
+				ExecutedQueryString: string(eq),
+			}
+
+			if link == "" {
+				continue
+			}
+
+			for _, field := range frame.Fields {
```

Comment:
i have tested, we don't need to add config to all field anymore, just Fields[1] would be good.

Reply:
that's great news! I'll update accordingly

Reply:
actually that didn't work for me @ying-jeanne